enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -71,6 +71,22 @@ scrape_configs:
       - source_labels: [__name__]
         regex: container_(network_tcp_usage_total|tasks_state|cpu_usage_seconds_total|memory_failures_total|network_udp_usage_total)
         action: drop
+      # extract workload from workload-hash-rand
+      - source_labels: [kubernetes_pod_name, es_workload]
+        regex: (.*)-[456789bcdf]{4,10}-[bcdfghjklmnpqrstvwxz2456789]{5};
```

Comment:
Just curious why no `123` in `[456789bcdf]`

Reply:
this is the SafeString set, full explanation in the issue.  The "hash" is from a string of digits, so there's only `[0-9]` which maps in SafeString to `[456789bcdf]`